third_party/Android.mk

```diff
@@ -104,218 +104,4 @@ LOCAL_C_INCLUDES:=$(GLSLANG_LOCAL_PATH) \
-        $(SPV_CORE11_GRAMMAR)
-		@$(HOST_PYTHON) $(SPVTOOLS_LOCAL_PATH)/utils/generate_grammar_tables.py \
-		                --spirv-core-grammar=$(SPV_CORE11_GRAMMAR) \
-		                --extension-enum-output=$(1)/extension_enum.inc \
-		                --enum-string-mapping-output=$(1)/enum_string_mapping.inc
-		@echo "[$(TARGET_ARCH_ABI)] Generate enum<->string mapping <= grammar JSON files"
-# Generated header extension_enum.inc is transitively included by table.h, which is
-# used pervasively.  Capture the pervasive dependency.
-$(foreach F,$(SPVTOOLS_SRC_FILES) $(SPVTOOLS_OPT_SRC_FILES),$(SPVTOOLS_LOCAL_PATH)/$F ) \
-  : $(1)/extension_enum.inc
-$(SPVTOOLS_LOCAL_PATH)/source/enum_string_mapping.cpp: $(1)/enum_string_mapping.inc
-endef
-$(eval $(call gen_spvtools_enum_string_mapping,$(SPVTOOLS_OUT_PATH)))
-
-define gen_spvtools_build_version_inc
-$(call generate-file-dir,$(1)/dummy_filename)
-$(1)/build-version.inc: \
-        $(SPVTOOLS_LOCAL_PATH)/utils/update_build_version.py \
-        $(SPVTOOLS_LOCAL_PATH)/CHANGES
-		@$(HOST_PYTHON) $(SPVTOOLS_LOCAL_PATH)/utils/update_build_version.py \
-		                $(SPVTOOLS_LOCAL_PATH) $(1)/build-version.inc
-		@echo "[$(TARGET_ARCH_ABI)] Generate       : build-version.inc <= CHANGES"
-$(SPVTOOLS_LOCAL_PATH)/source/software_version.cpp: $(1)/build-version.inc
-endef
-$(eval $(call gen_spvtools_build_version_inc,$(SPVTOOLS_OUT_PATH)))
-
-define gen_spvtools_generators_inc
-$(call generate-file-dir,$(1)/dummy_filename)
-$(1)/generators.inc: \
-        $(SPVTOOLS_LOCAL_PATH)/utils/generate_registry_tables.py \
-        $(SPVHEADERS_LOCAL_PATH)/include/spirv/spir-v.xml
-		@$(HOST_PYTHON) $(SPVTOOLS_LOCAL_PATH)/utils/generate_registry_tables.py \
-		                --xml=$(SPVHEADERS_LOCAL_PATH)/include/spirv/spir-v.xml \
-				--generator-output=$(1)/generators.inc
-		@echo "[$(TARGET_ARCH_ABI)] Generate       : generators.inc <= spir-v.xml"
-$(SPVTOOLS_LOCAL_PATH)/source/opcode.cpp: $(1)/generators.inc
-endef
-$(eval $(call gen_spvtools_generators_inc,$(SPVTOOLS_OUT_PATH)))
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := SPIRV-Tools
-LOCAL_C_INCLUDES := \
-		$(SPVTOOLS_LOCAL_PATH)/include \
-		$(SPVTOOLS_LOCAL_PATH)/source \
-		$(SPVTOOLS_LOCAL_PATH)/external/spirv-headers/include \
-		$(SPVTOOLS_OUT_PATH)
-LOCAL_EXPORT_C_INCLUDES := \
-		$(SPVTOOLS_LOCAL_PATH)/include
-LOCAL_CXXFLAGS:=-std=c++11 -fno-exceptions -fno-rtti -Werror
-LOCAL_SRC_FILES:= $(SPVTOOLS_SRC_FILES)
-include $(BUILD_STATIC_LIBRARY)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := SPIRV-Tools-opt
-LOCAL_C_INCLUDES := \
-		$(SPVTOOLS_LOCAL_PATH)/include \
-		$(SPVTOOLS_LOCAL_PATH)/source \
-		$(SPVTOOLS_LOCAL_PATH)/external/spirv-headers/include \
-		$(SPVTOOLS_OUT_PATH)
-LOCAL_CXXFLAGS:=-std=c++11 -fno-exceptions -fno-rtti -Werror
-LOCAL_STATIC_LIBRARIES:=SPIRV-Tools
-LOCAL_SRC_FILES:= $(SPVTOOLS_OPT_SRC_FILES)
-include $(BUILD_STATIC_LIBRARY)
+include $(THIRD_PARTY_PATH)/spirv-tools/Android.mk
```

Comment:
Use SPVTOOLS_LOCAL_PATH to find that Android.mk file.

We support placing SPIRV-Tools in places other than under third_party/spirv-tools